front/pages/api/w/[wId]/invitations/[iId]/index.ts

```diff
@@ -88,9 +94,10 @@ async function handler(
       }
       const body = bodyValidation.right;
 
-      invitation = await updateInvitationStatus(auth, {
-        invitation,
+      invitation = await updateInvitationStatusAndRole(auth, {
+        invitation: invitation,
         status: body.status,
+        role: body.initialRole as ActiveRoleType,
```

Comment:
You should not have to use `as`. Also this one is a lie since body.initialRole can be undefined as per above

Reply:
👍🏼